src/rpc/README.md

```diff
@@ -524,6 +471,72 @@ Cancels an invoice, only when invoice is in status `Open` can be canceled.
 * `status` - CkbInvoiceStatus, The invoice status
 
 
+<a id="payment"></a>
+### Module `Payment`
+RPC module for channel management.
+
+
+<a id="payment-send_payment"></a>
+#### Method `send_payment`
+
+Sends a payment to a peer.
+
+##### Params
+
+* `target_pubkey` - `Option<Pubkey>`, the identifier of the payment target
+* `amount` - `Option<u128>`, the amount of the payment
+* `payment_hash` - `Option<Hash256>`, the hash to use within the payment's HTLC
+* `final_tlc_expiry_delta` - `Option<u64>`, the TLC expiry delta should be used to set the timelock for the final hop, in milliseconds
+* `tlc_expiry_limit` - `Option<u64>`, the TLC expiry limit for the whole payment, in milliseconds, each hop is with a default tlc delta of 1 day
+ suppose the payment router is with N hops, the total tlc expiry limit is at least (N-1) days
+ this is also the default value for the payment if this parameter is not provided
+* `invoice` - `Option<String>`, the encoded invoice to send to the recipient
+* `timeout` - `Option<u64>`, the payment timeout in seconds, if the payment is not completed within this time, it will be cancelled
+* `max_fee_amount` - `Option<u128>`, the maximum fee amounts in shannons that the sender is willing to pay
+* `max_parts` - `Option<u64>`, max parts for the payment, only used for multi-part payments
+* `keysend` - `Option<bool>`, keysend payment
+* `udt_type_script` - `Option<Script>`, udt type script for the payment
+* `allow_self_payment` - `Option<bool>`, allow self payment, default is false
+* `hop_hints` - `Option<Vec>`, Optional route hints to reach the destination through private channels.
```

Comment:
the generated doc should be `Option<Vec<HopHint>>`, may be generator bug?

Reply:
yes, I will fix it in a separate pr.